doc/start/os-recommendations.rst

```diff
@@ -38,19 +38,27 @@ Platforms
 
 The charts below show how Ceph's requirements map onto various Linux
 platforms.  Generally speaking, there is very little dependence on
-specific distributions aside from the kernel and system initialization
+specific distributions outside of the kernel and system initialization
 package (i.e., sysvinit, systemd).
 
-+--------------+--------+--------+---------------+----------+--------+
-| Release Name | Tag    | CentOS | Ubuntu        | OpenSUSE | Debian |
-+==============+========+========+===============+==========+========+
-| Quincy       | 17.2.z | 8      | 20.04         | 15.3     | 11     |
-+--------------+--------+--------+---------------+----------+--------+
-| Pacific      | 16.2.z | 8      | 18.04, 20.04  | 15.2     | 10, 11 |
-+--------------+--------+--------+---------------+----------+--------+
-| Octopus      | 15.2.z | 7, 8   | 18.04, 20.04  | 15.2     | 10     |
-+--------------+--------+--------+---------------+----------+--------+
-
++--------------+--------+------------------------+---------------+----------+-----------------+
+| Release Name | Tag    | CentOS                 | Ubuntu        | OpenSUSE | Debian          |
++==============+========+========================+===============+==========+=================+
+| Quincy       | 17.2.z | 8 :sup:`A`             | 20.04         | 15.3     | 11 :sup:`C`     |
```

Comment:
A  applies to 20.04 for each ceph release. 18.04 is C at this point for pacific and octopus.

C applies to all opensuse and debian versions - maybe superscript the column header for those?

Reply:
I've shifted the column around so that it looks like this.